src/node/blockstorage.h

```diff
@@ -140,6 +140,8 @@ class BlockManager
     std::unordered_map<std::string, PruneLockInfo> m_prune_locks GUARDED_BY(::cs_main);
 
 public:
+    explicit BlockManager(const CBlockTreeDB::Options& block_tree_db_opts);
```

Comment:
Wouldn't this be better wrapped into a blockmanopts struct? (Happy to do it myself in another pull if you agree, since I need it elsewhere)

Reply:
Yeah agree, didn't do it here since this is the only opt, but feel free to do it in your pull!

Reply:
See https://github.com/bitcoin/bitcoin/pull/25781 :black_cat: